bpf/include/common.h

```diff
@@ -131,9 +131,14 @@ struct {
     __type(value, struct buf);
 } tmp_buf SEC(".maps");
 
+/*
+ * This map is used to store different configuration options:
+ * - key 0: Stores the log level
+ * - key 1: Stores the authz (authorization) toggle
+ */
 struct {
     __uint(type, BPF_MAP_TYPE_ARRAY);
-    __uint(max_entries, 1);
+    __uint(max_entries, 4);
```

Comment:
nit https://github.com/kmesh-net/kmesh/pull/1015/files#diff-e531188f2d6f84d9df52b52c90eb506619ef6d6d6ec4169dd093fcc33531a3fbR66 we are changing to a config map